covid/explore_data.py

```diff
@@ -0,0 +1,82 @@
+from collections import Counter
+import json
+import os
+
+import pandas as pd
+
+articles_dir = 'data/raw/CORD-19-research-challenge/'
+articles_folders = [
+    'biorxiv_medrxiv/biorxiv_medrxiv/pdf_json/',
+    'comm_use_subset/comm_use_subset/pdf_json/',
+    'comm_use_subset/comm_use_subset/pmc_json/',
+    'noncomm_use_subset/noncomm_use_subset/pdf_json/',
+    'noncomm_use_subset/noncomm_use_subset/pmc_json/',
+    'custom_license/custom_license/pdf_json/',
+    'custom_license/custom_license/pmc_json/'
+]
+
+def produce_metatada(articles_folders):
```

Comment:
spelling of data

Reply:
believe it or not, i was staring at this for a while before i understood the problem